src/mediaplayer/scss/player.scss

```diff
@@ -343,6 +343,32 @@ $controlsHeight: 36px;
                 bottom: 0;
             }
         }
+
+        &.stalled {
+            .player {
+                .player-overlay {
+                    [data-control="reload"] {
+                        margin-top: 0;
+
+                        display: flex;
+                        flex-direction: row;
+                        justify-content: center;
+
+                        &.reload {
+                            width: 100%;
+                            font-size: 20px;
+                            line-height: 20px;
+
+                            .message {
+                                font-size: 10px;
```

Comment:
Just a note: I assume this font size is fine for ux. It's very small, but I don't have suggestion for improvement.

Reply:
Actually, it needs to fit in the audio player space. So not so many choices unfortunately.